src/Microsoft.Health.Fhir.Tests.Common/TestFiles/Normative/SubscriptionForEncounter.json

```diff
@@ -0,0 +1,50 @@
+{
+    "resourceType": "Subscription",
```

Comment:
Since these files aren't normative they should live under /R4, I think it would be ok for the latest version (R5) to live here